docs/docs/amoro.md

```diff
@@ -0,0 +1,89 @@
+
+Lakehouse is characterized by its openness and loose coupling, with data and files maintained by users through various engines. While this
+architecture appears to be well-suited for T+1 scenarios, as more attention is paid to applying Lakehouse to streaming data warehouses and real-time
+analysis scenarios, challenges arise. For example:
+
+- Streaming writes bring a massive amount of fragment files
+- CDC ingestion and streaming updates generate excessive redundant data
+- Using the new data lake format leads to orphan files and expired snapshots.
+
+These issues can significantly affect the performance and cost of data analysis. Therefore, Amoro has introduced a Self-optimizing mechanism to
+create an out-of-the-box Streaming Lakehouse management service that is as user-friendly as a traditional database or data warehouse. Self-optimizing involves various procedures such as file compaction, deduplication, and sorting.
+
+The architecture and working mechanism of Self-optimizing are shown in the figure below:
+
+![Self-optimizing architecture](https://github.com/apache/amoro/blob/master/docs/images/concepts/self-optimizing_arch.png)
+
+The Optimizer is a component responsible for executing Self-optimizing tasks. It is a resident process managed by [AMS](https://amoro.apache.org/docs/latest/#architecture). AMS is responsible for
+detecting and planning Self-optimizing tasks for tables, and then scheduling them to Optimizers for distributed execution in real-time. Finally, AMS
+is responsible for submitting the optimizing results. Amoro achieves physical isolation of Optimizers through the Optimizer Group.
+
+The core features of [Amoro's Self-optimizing](https://amoro.apache.org/docs/latest/self-optimizing/) are:
+
+- Automated, Asynchronous and Transparent — Continuous background detecting of file changes, asynchronous distributed execution of optimizing tasks,
+  transparent and imperceptible to users
+- Resource Isolation and Sharing — Allow resources to be isolated and shared at the table level, as well as setting resource quotas
+- Flexible and Scalable Deployment — Optimizers support various deployment methods and convenient scaling
+
+
+# Iceberg Format
+
+Apache Amoro supports all catalog types supported by Iceberg, including common catalog: [REST](https://iceberg.apache.org/concepts/catalog/#decoupling-using-the-rest-catalog), Hadoop, Hive, Glue, JDBC, Nessie and other third-party catalog.
+Amoro supports all storage types supported by Iceberg, including common store: Hadoop, S3, GCS, ECS, OSS, and so on.
+
+At the same time, we also provide a unique form based on Apache Iceberg, including mixed-Iceberg Format and mixed-Hive Format, so that you can quickly upgrade to the iceberg+hive Mixed table while compatible with the original Hive data
+
+## Mixed-Iceberg Format
+
+[Mixed-Iceberg Format](https://amoro.apache.org/docs/latest/self-optimizing/) Compared with Iceberg format, Mixed-Iceberg format provides more features:
```

Comment:
The link seems wrong. I guess https://amoro.apache.org/docs/latest/mixed-iceberg-format/ is the correct link.